CHANGELOG.md

```diff
@@ -355,6 +390,14 @@ Additions to existing modules
   ```
 
 * In `Data.Fin.Properties`:
+
+  punchIn-mono-≤     : ∀ i (j k : Fin n) → j ≤ k → punchIn i j ≤ punchIn i k
+  punchIn-cancel-≤   : ∀ i (j k : Fin n) → punchIn i j ≤ punchIn i k → j ≤ k
+  punchOut-mono-≤    : (i≢j : i ≢ j) (i≢k : i ≢ k) → j ≤ k → punchOut i≢j ≤ punchOut i≢k
+  punchOut-cancel-≤  : (i≢j : i ≢ j) (i≢k : i ≢ k) → punchOut i≢j ≤ punchOut i≢k → j ≤ k
+  ```
+
+* Added new proofs in `Data.Integer.Properties`:
```

Comment:
Almost there!
The first block of `Data.Fin.Properties` is from https://github.com/agda/agda-stdlib/pull/1913 , while the second is about `Fin`, and *not* `Integer`, and comes from https://github.com/agda/agda-stdlib/pull/2748 ...
```suggestion

```

Reply:
That's embarrassing - I had fixed that locally, but not pushed! Properly fixed now.